app/apps/server/orchestrator.js

```diff
@@ -82,9 +82,10 @@ class AppServerOrchestrator {
 		return this._manager;
 	}
 
-	getEnabledGames() {
-		const { externalComponentManager: { providedComponents } } = this._manager;
-		return providedComponents;
+	getRegisteredExternalComponents() {
+		const { externalComponentManager: { registeredExternalComponents } } = this._manager;
```

Comment:
Move this to a method call. `this._manager.getExternalComponentManager().getProvidedComponents()`

Reply:
Resolved in https://github.com/RocketChat/Rocket.Chat/pull/15495/commits/78e4b264551c30da41bd8f1c3a4c363c03d9f994